web/static/pages/node_info/node-info.mjs

```diff
@@ -111,7 +111,7 @@ customElements.define('node-info',class NodeInfoElement extends LitElement {
                         <td>${task.ID}</td>
                         <td>${task.Task}</td>
                         <td>${task.Posted}</td>
-                        <td>${task.PoRepSector ? html`<a href="/pages/pipeline_porep/">f0${task.PoRepSectorSP}:${task.PoRepSector}</a>` : ''}</td>
+                        <td>${task.PoRepSector ? html`<a href="/pages/sector/?sp=f0${task.PoRepSectorSP}&id=${task.PoRepSector}">f0${task.PoRepSectorSP}:${task.PoRepSector}</a>` : ''}</td>
```

Comment:
Is that f0 compatible with calibrationNet?

Reply:
yep, parser doesn't care